app/index.js

```diff
@@ -32,6 +38,31 @@ const prisma = new PrismaClient();
 app.set('view engine', 'ejs'); // Config express to use ejs as the "view engine" (See: https://expressjs.com/en/guide/using-template-engines.html)
 app.set('views', './app/views'); // Config to use the views from our app dir
 
+app.use(session({
+    cookie: { maxAge: 2 * 60 * 60 * 1000 },
```

Comment:
I added an environmental variable called HTTPS_ENABLED so that we can disable it in dev. I wrote the code such that it defaults to true if the variable is not found.